src/main/java/com/safetynet/safetynetalerts/dao/JSONReader.java

```diff
@@ -1,4 +1,59 @@
 package com.safetynet.safetynetalerts.dao;
 
-public class JSONReader {
+import com.jsoniter.JsonIterator;
+import com.jsoniter.any.Any;
+import com.safetynet.safetynetalerts.model.FireStation;
+import com.safetynet.safetynetalerts.model.MedicalRecord;
+import com.safetynet.safetynetalerts.model.Person;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Slf4j
+@Component
+public class JSONReader implements Reader {
+    private Any initRead() throws IOException {
+        String input = "src/main/resources/data.json";
```

Comment:
I'd prefer to inject this, in order to be able to unit test this file more easily :)